visualization/app/codeCharta/ui/codeMap/threeViewer/threeSceneService.ts

```diff
@@ -160,6 +164,124 @@ export class ThreeSceneService implements CodeMapPreRenderServiceSubscriber {
 		}
 	}
 
+	hoverLabel(hoveredLabel: Object3D, raycaster: Raycaster, labels: Object3D[]) {
```

Comment:
The function animates the label, so we could rename the function to `animateLabelb(hoveredLabel...)`. This would make clear what the complicate calculations are doing (I guess).

Reply:
where is the <b>b</b>? I really can't see what you mean?

Reply:
In my suggestion only :)